.github/workflows/pr.yml

```diff
@@ -14,13 +14,31 @@ env:
   PARALLEL: -j 2
 
 jobs:
-  mkl:
-    runs-on: intel-ubuntu-latest
+  unit-tests:
+    runs-on: ubuntu-latest
     # One runner for each domain
     strategy:
       matrix:
-        domain: [blas, dft, lapack, rng]
-    name: MKL ${{ matrix.domain }} CPU
+        include:
+        - config: portBLAS
+          options: -DENABLE_PORTBLAS_BACKEND=ON -DENABLE_MKLCPU_BACKEND=OFF -DTARGET_DOMAINS=blas -DPORTBLAS_TUNING_TARGET=INTEL_CPU
+          tests: '.*'
+        - config: portFFT
+          options: -DENABLE_PORTFFT_BACKEND=ON -DENABLE_MKLCPU_BACKEND=OFF -DTARGET_DOMAINS=dft -DCMAKE_CXX_FLAGS="-fsycl -fsycl-targets=spir64"
+          tests: 'DFT/CT/.*ComputeTests_in_place_COMPLEX.COMPLEX_SINGLE_in_place_buffer.sizes_8_batches_1*'
+        - config: MKL BLAS
+          options: -DTARGET_DOMAINS=blas
```

Comment:
This is a minor improvement to avoid CMake warnings. Since now we separate domain options, we may move `-DREF_BLAS_ROOT` here and `-DREF_LAPACK_ROOT` in `MKL LAPACK`.

Reply:
moved...